### PR TITLE
Update using samples

### PR DESCRIPTION
**Pull request by ayshsandu**

### CODE REVIEW

File: en/docs/develop/customizing-login-pages-for-service-providers.md [hunk 1]
```diff
@@ -29,39 +29,42 @@ SAML SSO service providers that send authentication requests to it. The
...
 
-1.  [Deploy traveloctity](../../develop/deploying-the-sample-app/) sample application.
+1.  [Deploy traveloctity](../../develop/deploying-the-sample-app/#deploying-travelocity-webapp) sample application.
 
-    !!! note
-        - Open `           travelocity.properties          ` file and check for following 
+2.  Get a copy of `travelocity.com.war` that
+    [download from here](../../develop/deploying-the-sample-app/#deploy-the-sample-web-application)
+    to the same location `<TOMCAT_HOME>/webapps/` rename as
+    `acis.com.war`. Restart the tomcat server.
```
Comment (sherenewso2) [inline en/docs/develop/customizing-login-pages-for-service-providers.md]: change "acis" to "avis"?

File: en/docs/develop/customizing-login-pages-for-service-providers.md [hunk 2]
```diff
@@ -29,39 +29,42 @@ SAML SSO service providers that send authentication requests to it. The
...
             ```
             #The URL of the SAML 2.0 Assertion Consumer
             SAML2.AssertionConsumerURL=http://wso2.is:8080/avis.com/home.jsp
         
             #openid.return_to parameter
             OpenId.ReturnToURL=http://wso2.is:8080/avis.com/home.jsp
             ```
-
-2.  Start the application server and access following URLs to make sure
+3. Register a new service provider with name `avis.com` simillerly as
```
Comment (sherenewso2) [inline en/docs/develop/customizing-login-pages-for-service-providers.md]: simillerly --> similarly

File: en/docs/learn/configuring-openid-connect-back-channel-logout.md [hunk 3]
```diff
@@ -31,13 +31,31 @@ back-channel logout.
...
+
+### Configuring the sample applications
+
+1. Follow the steps
+   [here](../../develop/deploying-the-sample-app/#deploying-playground2-webapp)
+   to download, deploy and register `playground2` applications.
+
+
+2.  Make a copy of ` playground2.war `it and rename to `
+    playground3.war` in the same location described in step 1.
```
Comment (sherenewso2) [inline en/docs/learn/configuring-openid-connect-back-channel-logout.md]: shall we remove the "it" in this sentence:

 "Make a copy of playground2.war **it** and rename to...."

File: en/docs/develop/customizing-login-pages-for-service-providers.md (hunk 1 above)
Comment (ayshsandu) [reply en/docs/develop/customizing-login-pages-for-service-providers.md]: Fixed

File: en/docs/develop/customizing-login-pages-for-service-providers.md (hunk 2 above)
Comment (ayshsandu) [reply en/docs/develop/customizing-login-pages-for-service-providers.md]: Fixed

File: en/docs/learn/configuring-openid-connect-back-channel-logout.md (hunk 3 above)
Comment (ayshsandu) [reply en/docs/learn/configuring-openid-connect-back-channel-logout.md]: Fixed

**Pull request merged**